examples/src/main/python/ml/kmeans_example.py

```diff
@@ -17,55 +17,43 @@
 
 from __future__ import print_function
 
-import sys
-
-import numpy as np
+# $example on$
 from pyspark.ml.clustering import KMeans, KMeansModel
-from pyspark.mllib.linalg import VectorUDT, _convert_to_vector
+# $example off$
+
 from pyspark.sql import SparkSession
-from pyspark.sql.types import Row, StructField, StructType
 
 """
 A simple example demonstrating a k-means clustering.
 Run with:
-  bin/spark-submit examples/src/main/python/ml/kmeans_example.py <input> <k>
-
-This example requires NumPy (http://www.numpy.org/).
```

Comment:
So I believe this example still requires NumPy even though it isn't explicitly imported (see inside of `def toArray` called inside of `clusterCenters` which says it returns a NumPy array).